operator/bundle/manifests/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -36,22 +36,22 @@ spec:
       name: centrals.platform.stackrox.io
       resources:
       - kind: Deployment
-        name: central
+        name: ''
         version: v1
       - kind: Route
-        name: central
+        name: ''
         version: v1
       - kind: Secret
-        name: central-htpasswd
+        name: ''
         version: v1
       - kind: Service
-        name: central-loadbalancer
+        name: ''
         version: v1
       - kind: Deployment
-        name: scanner
+        name: ''
         version: v1
       - kind: Deployment
-        name: scanner-db
+        name: ''
```

Comment:
@cchen666 What's the purpose to declare owned resources if what we see here says: "this operator owns one Route, one Secret, one Service and three Deployments"?

Reply:
Hi Misha, I think I need to remove duplicate Deployment entires and I also need to take into account for SecuredCluster CRD.